### PR TITLE
[Term Entry] Kotlin Strings .prependIndent()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the .prependIndent() method for Kotlin Strings
- Closes Issue #3731   

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.prependIndent()`** method in Kotlin is used to add indentation to a string. It is useful for formatting text, especially when we need to align or indent lines within a string.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md:15]:
```suggestion
The **`.prependIndent()`** method in Kotlin is used to add indentation to a `string`. It is useful for formatting text, especially when we need to align or indent lines within a `string`.
```

File: content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+This example shows how to use the `.prependIndent()` method to add indents to a string:
+
+```kotlin
+fun main() {
+    val originalText = "'This is an indented string.'"
+
+    // Adds three spaces of indentation to the beginning of the string
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md:34]:
```suggestion
```
Let's remove it from here and add this in the intro of this code sample

File: content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```pseudo
+fun String.prependIndent(indent: String = " "): String
+```
+
+- `String`: The string we are adding indentation to.
+- `indent: String`: The indentation we want to add to the string.
+
+## Example
+
+This example shows how to use the `.prependIndent()` method to add indents to a string:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md:28]:
```suggestion
This example shows how to use the `.prependIndent()` method to add three spaces of indentation to the beginning of the `string`:
```

File: content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+fun main() {
+    val originalText = "'This is an indented string.'"
+
+    // Adds three spaces of indentation to the beginning of the string
+    val indentedText = originalText.prependIndent("  ")
+    println(indentedText)
+}
+```
+
+The code will return the following output:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/prependIndent/prependIndent.md:40]:
```suggestion
The above code will return the following output:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Nice entry 😊 Here are few requested changes from my side

**Comment by dakshdeepHERE** [comment]
@Fejiro001 This look good for 2nd review.

**Review by dakshdeepHERE — APPROVED** [review]

**Review by SSwiniarski — APPROVED** [review]
@Fejiro001 I made a slight edit ro remove first person pronouns, now it looks good to merge.

**Pull request merged**